signer/src/message.rs

```diff
@@ -165,6 +165,8 @@ pub struct StacksTransactionSignRequest {
     /// It's essentially a hash of the contract call struct, the nonce, the
     /// tx_fee and a few other things.
     pub digest: [u8; 32],
+    /// The transaction ID of the associated contract call transaction.
+    pub txid: blockstack_lib::burnchains::Txid,
```

Comment:
Is this docstring/type correct? What associated contract call tx (on bitcoin?) are we referring to?

Reply:
Yeah I believe this is correct. Maybe we should use a type alias or the internal `StacksTxId` type here to make that clear, but `blockstack_lib::burnchains::Txid` is the type for transaction IDs on stacks. See:
https://github.com/stacks-network/stacks-core/blob/2e67fa900b5a1ef9f7268529b9c2d1fdcc72c5d4/stackslib/src/chainstate/stacks/transaction.rs#L824-L829
and in that same file:
https://github.com/stacks-network/stacks-core/blob/2e67fa900b5a1ef9f7268529b9c2d1fdcc72c5d4/stackslib/src/chainstate/stacks/transaction.rs#L34-L36
.